k8s/controllers/k8s/backup_controller.go

```diff
@@ -54,8 +54,8 @@ type FailoverConfig struct {
 
 // BackupConfig holds all configuration about backup
 type BackupConfig struct {
-	Name  string
-	Limit string
+	Name      string
+	ObjectIDs string
```

Comment:
Use list of string if you would like to pass multiple value.

Reply:
Okay. That's good.

Reply:
Actually, I think it's better to make this consistent between vineyardctl and CRD to reduce the type convert. WDUT? @sighingnow 
https://github.com/v6d-io/v6d/pull/1402/files#diff-08fb2efc269a72bc81aedac5426c3b1cec120baae20021fa81beb33fcf0e9d68R52-R54

Reply:
See https://pkg.go.dev/github.com/spf13/pflag#StringSlice

cobra support list arguments as well (via pflags). Don't use `,` or `;` to reinvent your own list stringify/encoding schemas.

Reply:
Thanks, make sense to me.